person.rb

```diff
@@ -7,9 +8,10 @@ def initialize(age:, parent_permission: true, name: 'Unknown')
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
```

Comment:
- [Optional] Dear @ZewdieMc, It's a common practice to have our getter and setters next to the class declaration before the `initialize` method. This is to preserve code readability and improve collaboration by following common practices.

Reply:
Dear @TadesseDev ,
Thank you for your important feedback.